railsconf/2020/let_juniors_fail.md

```diff
@@ -0,0 +1,36 @@
+# Give you juniors space to fail/Benefits of allowing junior developers to fail
+
+## Abstract
+
+In this talk, we will discuss the many different ways you can create an environment that promotes creativity and experimentation for junior developers. To create an environment that fosters mentorship, rapid growth, and respect, you must be willing to let your junior developers fail. Inversely, if you are a junior developer, you should seek out the type of environments that will not only allow, but encourage you to push push boundaries and learn from the resulting success or failure. This talk will focus on how to be a good mentor, and how to walk with your juniors instead of in front of them by learning to harness the unchecked optimism of your junior developers.
```

Comment:
... I will present several ways in which you can create

Reply:
Such an environment must encourage mentorship

Reply:
This is possible only if you create space for your junior developers to fail with dignity

Reply:
If you are a junior developer yourself, your mission is to seek out environments that will encourage you to push your boundaries without equating failure with humiliation.

Reply:
"how to walk with your juniors instead of leading them by the ear." (I would omit the rest of the sentence.)